packages/runner-ct/src/lib/webpack-hmr-client.ts

```diff
@@ -30,12 +36,14 @@ export function connectWebpackHmr ({ url, onReload }) {
 
       onReload()
     },
-    errors: (errors) => {
+    errors: (errors: string[]) => {
       console.error('Errors while compiling. Reload prevented.')
 
       errors.forEach(
         (warning) => console.warn(stripAnsi(warning)),
       )
+
+      eventManager.emit('script:error', { error: errors[0] })
```

Comment:
I guess showing the first error is fine. We could do `join` to show them all, but it look like `<ScriptError>` is designed to just show one error, not many.

Reply:
I've seen multiple errors before (Vite??), but... this is sufficient for webpack's model. We generally only care about transpile failing and that's a single process.exit anyway